Sources/iPad/View Controllers/CloudSignup/Welcome/WelcomeViewController_iPad.m

```diff
@@ -117,6 +118,10 @@ - (void)willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrie
 - (void) didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation
 {
     scrollingLocked = NO;
+    if ([AppDelegate_iPad instance] && [AppDelegate_iPad instance].rootViewController){
+        [[AppDelegate_iPad instance].rootViewController didRotateFromInterfaceOrientation:fromInterfaceOrientation];
+    }
+
```

Comment:
This bloc is only use to force the orientation when the welcome VC is displaying